File: GSI QA testing tool/UI/UI_170_ClaimConfirmation.cs
using OpenQA.Selenium.Support.UI;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;
using GSI_QA_testing_tool.Utilities;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_170_ClaimConfirmation
    {
        public static void GoTo(IWebDriver driver)
        {

            if (Data._StopAt.Contains("Claim Confirmation"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }




            if (Finder.FindIt(driver, "//input[@id='ctl00_Main_content_btnCertifyClaim']"))
            {
                Debug.WriteLine("ClaimConfirmation is On");
            }
            else
            {
                Debug.WriteLine("ClaimConfirmation is Off");
                return;
            }


            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnCertifyClaim']");
        }
    }
}

File: GSI QA testing tool/UI/UI_170_ClaimConfirmation.vb
Imports OpenQA.Selenium.Support.UI
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_170_ClaimConfirmation
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Claim Confirmation") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "#ctl00_Main_content_btnCertifyClaim") Then
                Debug.WriteLine("ClaimConfirmation is On")
            Else
                Debug.WriteLine("ClaimConfirmation is Off")
                Return
            End If

            Finder.JSClickIt(driver, "#ctl00_Main_content_btnCertifyClaim", js)
        End Sub
    End Class
End Namespace
